src/util/heap/layout/byte_map_mmapper.rs

```diff
@@ -206,7 +206,7 @@ mod tests {
     use crate::util::heap::layout::vm_layout_constants::MMAP_CHUNK_BYTES;
     use std::sync::atomic::Ordering;
 
-    const MEGABYTE: usize = 1 << 20;
+    const MEGABYTE: usize = 1 << 22;
```

Comment:
This does not make sense? You can rename it to `CHUNK_SIZE`.

Reply:
Done